cluster/src/main/scala/io/snappydata/gemxd/SparkSQLExecuteImpl.scala

```diff
@@ -483,7 +483,7 @@ object SnappyContextPerConnection {
     val context = connectionIdMap.get(connectionID)
     if (context != null) context
     else {
-      val snc = SnappyContext.getOrCreate(null: SparkContext).newSession()
+      val snc = SnappyContext(null: SparkContext).newSession()
```

Comment:
Yes, the tab should also list the username used for connection (later also the clientID esp for thrift connections which provide that info). That is a more imp issue to fix to make the UI usable.

How many concurrent connections are we expecting that route queries? We can suppress tabs that have not fired anything for a while.